public/index.html

```diff
@@ -32,6 +32,9 @@
     <script src="./injectEnv.js"></script>
     <script src="./js/session.js"></script>
 
+    <!-- Import web component -->
+    <script src="https://cbiit.github.io/nci-softwaresolutions-elements/components/include-html.js"></script>
```

Comment:
@rana22 looks like url is wrong. Correct url: "https://cbiit.github.io/crdc-alert-elements/components/include-html.js"